### DISCUSSION TITLE
Migrating to 2.4

### DISCUSSION THREAD

**Discussion by rosswakelin** [Security-Onion-Solutions/securityonion#10257] [2.4] [answered]
I'm looking into my cloudy crystal ball and thinking about how I'm going to upgrade my distributed (physically as well as logically) grid.  I have a bunch of new sensors to deploy around the country, and I was wondering if it is possible to install the current 2.3.2xx train of software on top of Rocky instead of Ubuntu or Centos 7.  This would make it possible for me to deploy these new devices without having to worry about underlying linux re-install at remote locations.

**Comment by dougburks** [comment, marked as answer]
Security Onion 2.3 only supports CentOS 7 and Ubuntu 20.04:
https://docs.securityonion.net/en/2.3/installation.html#installation-on-ubuntu-or-centos
